flang/runtime/command.cpp

```diff
@@ -14,6 +14,11 @@
 #include "flang/Runtime/descriptor.h"
 #include <cstdlib>
 #include <limits>
+#ifdef _WIN32
```

Comment:
I see that _WIN32 is defined for both WIndows 32 and 64 bit platforms, so this is fine.
https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170&redirectedfrom=MSDN